src/common/config_opts.h

```diff
@@ -1235,6 +1235,11 @@ OPTION(rbd_journal_object_flush_bytes, OPT_INT, 0) // maximum number of pending
 OPTION(rbd_journal_object_flush_age, OPT_DOUBLE, 0) // maximum age (in seconds) for pending commits
 OPTION(rbd_journal_pool, OPT_STR, "") // pool for journal objects
 
+/**
+ * RBD Mirror options
+ */
+OPTION(rbd_mirror_sync_point_update_interval, OPT_DOUBLE, 1) // number of seconds between each update of the image sync point object number
```

Comment:
Minor: can we switch "interval" to "age" and bump the default to something like 30 seconds?